loafwallet/TransactionsViewController.swift

```diff
@@ -57,6 +57,10 @@ class TransactionsViewController: UIViewController, UITableViewDelegate, UITable
     override func viewDidLoad() {
       setup()
       addSubscriptions()
+
```

Comment:
Yep ...Reviewing the class I think this was my hack to trigger the recommend rescan when hashrate spikes.
I own it.